Formula/s/sevenzip.rb

```diff
@@ -25,27 +25,38 @@ class Sevenzip < Formula
   end
 
   def install
+    mac_suffix = Hardware::CPU.intel? ? "x64" : Hardware::CPU.arch
+    mk_suffix, directory = if OS.mac?
+      ["mac_#{mac_suffix}", "m_#{mac_suffix}"]
+    else
+      ["gcc", "g"]
+    end
     cd "CPP/7zip/Bundles/Alone2" do
-      mac_suffix = Hardware::CPU.intel? ? "x64" : Hardware::CPU.arch
-      mk_suffix, directory = if OS.mac?
-        ["mac_#{mac_suffix}", "m_#{mac_suffix}"]
-      else
-        ["gcc", "g"]
-      end
-
       system "make", "-f", "../../cmpl_#{mk_suffix}.mak", "DISABLE_RAR_COMPRESS=1"
 
       # Cherry pick the binary manually. This should be changed to something
       # like `make install' if the upstream adds an install target.
       # See: https://sourceforge.net/p/sevenzip/discussion/45797/thread/1d5b04f2f1/
       bin.install "b/#{directory}/7zz"
     end
+    cd "CPP/7zip/Bundles/Format7zF" do
+      system "make", "-f", "../../cmpl_#{mk_suffix}.mak", "DISABLE_RAR_COMPRESS=1"
+      if OS.mac?
+        lib.install "b/#{directory}/7z.so" => "7z.dylib"
+      else
+        lib.install "b/#{directory}/7z.so"
+      end
```

Comment:
```suggestion
      lib.install "b/#{directory}/7z.so" => shared_library("7z")
      lib.install_symlink shared_library("7z") => shared_library("lib7z")
```
The symlink will allow you to link this with `-l7z`.